### PR TITLE
Create settings page for admin

### PR DESCRIPTION
**Pull request by 1Artemida**

### CODE REVIEW

File: client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+const AdminSettings = () => {
+
+  const dispatch = useDispatch();
+
+  const user = useSelector(userSelector);
+
+  const [isShowChanges, setShowChanges] = useState(false);
+  
+  const [firstName, setFirstname] = useState('');
```
Comment (MattMsh) [inline client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx]: in firstName and lastName you can get data from redux, like:
```suggestion
  const [firstName, setFirstname] = useState(user.firstName);
```

File: client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                  <span className={style.inputs_title}>First name</span>
+                  <input
+                    defaultValue={user.firstName}
+                    className={style.username_input}
+                    onChange={(event) => setFirstname(event.target.value)}
+                  >
+                  </input>
+                  <span className={style.inputs_title}>Last name</span>
+                  <input
+                    defaultValue={user.lastName}
```
Comment (MattMsh) [inline client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx]: here, as defaultValue you can use lastName from component's own state

File: client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
+import { React, useState }  from 'react';
+import routes from '../../../../configs/routes';
+import { userSelector } from '../../../../redux';
+import { Link } from 'react-router-dom';
+import resset_password_icon from '../../../../assets/images/SettingsPageImages/resset_password_image.svg';
+import username_icon from '../../../../assets//images/SettingsPageImages/username-icon.svg';
+import changes_was_saved_icon from '../../../../assets/images/SettingsPageImages/changes_was_saved.svg';
+import cancel from '../../../../assets/images/SettingsPageImages/cancel.svg';
+import style from './AdminSettings.module.scss';
+import SaveButton from '../../../../shared/Buttons/Save';
```
Comment (MattMsh) [inline client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx]: You can import SaveButton from Buttons, like: 
```suggestion
import { Save } from '../../../../shared/Buttons';
```

**Review by MattMsh — COMMENTED** [review]
It's empty file, if it is unusable, should be deleted
 client/src/assets/images/SideBarImages/account

File: client/src/routes/route.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import CreatingProject from '../pages/CreatingProjectPage';
 import AuthRoute from './AuthRoute';
 import UnauthenticatedRoute from './UnauthenticatedRoute';
 import './routes.scss';
-import AdminSettings from '../pages/SettingsPage/components/AdminSettingsPage/AdminSettings';
+import Settings from '../pages/SettingsPage/components/AdminSettingsPage/Settings';
```
Comment (MattMsh) [inline client/src/routes/route.js]: '../pages/SettingsPage/components/AdminSettingsPage/Settings' file doesn't exist
Route should be corrected

**Review by Vadym342 — APPROVED** [review]

**Review by Bagdunn — APPROVED** [review]

**Review by MattMsh — APPROVED** [review]

**Pull request merged**